pkg/tsdb/prometheus/buffered/time_series_query.go

```diff
@@ -155,6 +156,12 @@ func (b *Buffered) runQuery(ctx context.Context, query *PrometheusQuery) (backen
 	if query.RangeQuery {
 		rangeResponse, _, err := b.client.QueryRange(ctx, query.Expr, timeRange)
 		if err != nil {
+			var promErr *apiv1.Error
```

Comment:
@kylebrandt recently changed how instant and range queries collect errors for the new client in https://github.com/grafana/grafana/pull/57698. should something similar be done here?

Reply:
We will eventually deprecate buffered client so I do not want to spend too much time on this. But I will check what I can do. Thanks for the suggestion.